props/common.props

```diff
@@ -2,10 +2,32 @@
 <Project>
 
   <PropertyGroup>
-    <Description>Qowaiv implements common, universal domain objects. These types form the base of your domain model.</Description>
+    <Title>Qowaiv (Single) Value Object library</Title>
+    <Description>
+      Qowaiv is a (Single) Value Object library. It aims to model reusable (Single)
+      Value Objects that can be used a wide variety of modeling scenarios, both
+      inside and outside a Domain-driven context.
+    </Description>
     <Authors>Ad van der Hoeven;Corniel Nobel;Erik Ammerlaan;Fons Sonnemans;Jack Kester;Hanif Hafez Nezami;Laura Kramer;Patrick Evers;Vincent Lesierse;Wilko Frieke</Authors>
     <Owners>Qowaiv community</Owners>
-    <PackageTags>qowaiv domain model</PackageTags>
+    <PackageTags>
```

Comment:
"yes no", "country", "identifiers", "security", "serialization", "fluent", "clock"
You can also add tags related to the type of SVO's you have, so: chemistry, financial, globalization, mathematical etc. Or is that not specific enough?

Reply:
I'm not sure. I like the idea, but should we hope/expect that people searching for those terms find this package?

Reply:
Hmm, more tags is more visibility, but if they are too generic, Qowaiv will not stand out from the many results.